src/main/java/gameframework/drawing/GameCanvasDefaultImpl.java

```diff
@@ -70,6 +73,10 @@ public void addTo(Frame frame) {
 		frame.add(canvas);
 	}
 
+	public InputManager getInputManager() {
```

Comment:
you may want to add that to the interface `GameCanvas` as well because this is the return type of `GameData.getCanvas()`.